source/software/module_system_basics.rst

```diff
@@ -0,0 +1,343 @@
+be associated with either the same toolchain or compatible (sub)toolchains
+(see also https://docs.easybuild.io/common-toolchains/#toolchains_diagram).
+
+For example, once you have loaded a module that uses the ``foss/2023a``
+toolchain, all other modules that you load next should have been installed
+with the same toolchain or with compatible (sub)toolchains such as
+``GCCcore/12.3.0``, ``GCC/12.3.0`` and ``gompi/2023a``. Subtoolchains
+compatible with e.g. ``intel/2023a`` include ``GCCcore/12.3.0``,
+``intel-compilers/2023.1.0`` and ``iimpi/2023a``.
+
+Additionally, two versions of the same software packages can not be loaded
+together. If you e.g. loaded a ``Python/3.11.3-GCCcore-12.3.0`` module, then
+also loading another ``Python`` module (either directly or as a dependency of
+another module) will cause ``Python/3.11.3-GCCcore-12.3.0`` to be unloaded and
+replaced by the new module (the same will happen to the modules which both
+``Python`` modules load as dependencies).
+
+
+List loaded modules
+~~~~~~~~~~~~~~~~~~~
+
+Obviously, the user needs to keep track of the modules that are
+currently loaded. After executing the above load command, the list
+of loaded modules will look similar to:
+
+::
+
+   $ module list
+   Currently Loaded Modulefiles:
+     1) cluster/wice/batch
+     2) GCCcore/10.3.0
+     ...
+     16) OpenMPI/4.1.1-GCC-10.3.0
+     17) OpenBLAS/0.3.15-GCC-10.3.0
+     ...
+     46) PLUMED/2.9.0-foss-2023a
+     47) CP2K/2023.1-foss-2023a
+     48) GROMACS/2023.3-foss-2023a-PLUMED-2.9.0
+
+Note that this does not just show the two requested modules, but also all
+the modules that got loaded automatically in order to satisfy (runtime)
+dependencies of the explicitly loaded ``CP2K`` and ``GROMACS`` installations
+(``PLUMED``, ``OpenMPI``, ``OpenBLAS``, etcetera).
+
+
+Unloading modules
+~~~~~~~~~~~~~~~~~
+
+To unload a specific module, use the ``module unload`` command, e.g.:
+
+::
+
+   $ module unload CP2K
+
+Notice that the version was not specified: the module system is
+sufficiently clever to figure out what the user intends. However,
+checking the list of currently loaded modules is always a good idea,
+just to make sure.
+
+Keep in mind that ``module unload`` only unloads the chosen module.
+It will not unload other modules which were automatically loaded
+as dependencies of the chosen module.
+
+
+.. _module_purge:
+
+Purging modules
+~~~~~~~~~~~~~~~
+
+In order to unload all modules at once and start with a clean slate, use:
+
+::
+
+   $ module purge
+
+This will not unload so-called `sticky modules
+<https://lmod.readthedocs.io/en/latest/240_sticky_modules.html>`__, which
+are special modules that do not normally need to be unloaded (for example
+because they define the appropriate module paths and possibly other environment
+variables). If really needed, sticky modules can be unloaded with
+``module --force purge``.
+
+
+.. _collections of modules:
+
+Collections of modules
```

Comment:
see my remark above about module collections

Reply:
(addressed in 9c65cb57b60b63c34e1fe94e6f743915b22dbaef)